### PR TITLE
DO-284: Added implementation of birthday and anniversary.

### PR DESCRIPTION
**Pull request by ayushmalik2110**

### CODE REVIEW

File: src/utils/member/utils.js [hunk 1]
```diff
@@ -772,3 +774,22 @@ export const isFamilyDetailsDataDirty = (formData = [], updatedFormData = []) =>
  */
 export const removeUrlParamsFromPathname = pathname =>
   pathname.substr(0, pathname.lastIndexOf('/'));
+
+/**
+ * This util function check that if date is today by checking date and month.
+ * @param {String} dateToCompare
+ * @return {boolean}
+ */
+export const isDateToday = (dateToCompare = '') => {
```
Comment (mohitsaladiya) [inline src/utils/member/utils.js:783]: Move this to common > dateUtils.

File: src/components/common/CardView/CardImage/CardImage.jsx [hunk 2]
```diff
@@ -22,18 +26,28 @@ const CardImage = (props) => {
     <div onClick={onClickProfileImage} style={{ imageContainerStyle }} className="card-image">
       <div className="card-image-container">
         <img className="card-image_img" style={imageStyle} src={imageUrl} alt="card-image" />
```
Comment (mohitsaladiya) [inline src/components/common/CardView/CardImage/CardImage.jsx:28]: What about the desktop view?

File: src/utils/member/utils.js (hunk 1 above)
Comment (ayushmalik2110) [reply src/utils/member/utils.js]: Done.

File: src/components/common/CardView/CardImage/CardImage.jsx (hunk 2 above)
Comment (ayushmalik2110) [reply src/components/common/CardView/CardImage/CardImage.jsx:29]: Done

File: src/utils/member/utils.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import { isEmpty, isEqual } from 'lodash';
+import moment from 'moment';
```
Comment (mohitsaladiya) [inline src/utils/member/utils.js:2]: Remove this import if its not in use.

Comment (ayushmalik2110) [reply src/utils/member/utils.js]: Done

**Pull request merged**